src/components/ui/Navigation/Navigation.module.scss

```diff
@@ -0,0 +1,171 @@
+.container {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: var(--navbar-height);
+  z-index: 1;
+  background: var(--color-white);
+  box-shadow: 0 0 8px 2px rgb(0 0 0 / 33%);
+}
+
+.innerContainer {
+  display: flex;
+  align-items: center;
+  justify-content: stretch;
+  margin: auto;
+  padding: var(--navbar-padding-vertical) var(--navbar-padding-horizontal);
+  max-width: var(--section-max-width);
+}
+
+.logoWrapper {
+  width: var(--navbar-logo-size);
+  height: var(--navbar-logo-size);
+}
+
+.logo {
+  width: var(--navbar-logo-size);
+  height: var(--navbar-logo-size);
+}
+
+.linksWrapper {
+  width: 100vw;
+  height: 100vh;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: var(--color-white);
+  transform-origin: top;
+  transform: scaleY(0);
+  transition: transform ease-in-out 0.25s;
+}
+
+.links {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+  gap: 24px;
+  opacity: 0;
+  transition: opacity 0.15s ease-in-out;
+}
+
+.link {
+  text-decoration: none;
+  color: var(--color-black);
+  font-size: var(--l-font-size);
+  transition: 0.25s color ease-in-out;
+
+  &:hover,
+  &.active {
+    color: var(--color-orange);
+  }
+}
+
+.hamburger {
+  display: none;
+}
+
+%hamburgerTriggerBar {
+  background: var(--color-blue);
+  position: relative;
+  display: block;
+  height: var(--navbar-hamburger-bar-height);
+  transition:
+    transform 0.2s ease-out 0s,
+    background-color 0.2s ease-out 0.2s,
+    top 0.2s ease-out 0.3s;
+  width: var(--navbar-hamburger-size);
+}
+
+%hamburgerTriggerBarChecked {
+  transition:
+    top 0.2s ease-out 0s,
+    background-color 0s ease 0.2s,
+    transform 0.2s ease-out 0.3s;
+}
+
+.hamburgerTrigger {
+  cursor: pointer;
+  margin: auto 0 auto auto;
+  padding: 20px 8px;
+
+  span {
+    @extend %hamburgerTriggerBar;
+
+    &::before {
+      @extend %hamburgerTriggerBar;
+
+      content: "";
+      top: var(--navbar-hamburger-bottom-bar-top-position);
+    }
+
+    &::after {
+      @extend %hamburgerTriggerBar;
+
+      content: "";
+      top: var(--navbar-hamburger-top-bar-top-position);
+    }
+  }
+}
+
+.hamburger:checked {
+  & ~ .linksWrapper {
+    transform: scale(1);
+
+    & .links {
+      opacity: 1;
+      transition: opacity 0.25s ease-in-out 0.25s;
+    }
+  }
+
+  & ~ .hamburgerTrigger span {
+    background: transparent;
+
+    @extend %hamburgerTriggerBarChecked;
+
+    &::before {
+      @extend %hamburgerTriggerBarChecked;
+
+      top: 0;
+      transform: rotate(45deg);
+    }
+
+    &::after {
+      @extend %hamburgerTriggerBarChecked;
+
+      top: -2px;
```

Comment:
I'm not 100% sure but I guess we need to have `-3px` here to have it evenly spaced.

With -2px:
<img width="310" alt="image" src="https://user-images.githubusercontent.com/45982530/187630328-57d5c1ed-9d91-4f4b-8677-f263e0125e05.png">

With -3px: 
<img width="374" alt="image" src="https://user-images.githubusercontent.com/45982530/187630415-3363d2ce-4a80-47a1-a2fa-cb36314002c8.png">

Reply:
Definitely! Thanks for spotting that one! After deeper analysis - I've decided to adjust it depending on viewport, cause that's when the size is changing as little. Here are the results:

mobile

![mobile](https://user-images.githubusercontent.com/20808724/187728492-0ac9d36b-0d8e-499f-b9c1-d8aca0412cf1.png)

tablet

![tablet](https://user-images.githubusercontent.com/20808724/187728495-8e7d69c8-cb1b-44f7-a44f-2f01ad6e5106.png)